spec/fixtures/trade_leads/canada/canada_leads.csv

```diff
@@ -1,4 +1,4 @@
-"language","title","reference_number","solicitation_number","amendment_number","publication_date","date_closing","amendment_date","publishing_status","gsin","region_opportunity","region_delivery","notice_type","trade_agreement","tendering_procedure","competitive_procurement_strategy","non_competitive_procurement_strategy","procurement_entity","end_user_entity","description","access_terms_of_use","contact","document","attachment"
+﻿"language","title","reference_number","solicitation_number","amendment_number","publication_date","date_closing","amendment_date","publishing_status","gsin","region_opportunity","region_delivery","notice_type","trade_agreement","tendering_procedure","competitive_procurement_strategy","non_competitive_procurement_strategy","procurement_entity","end_user_entity","description","access_terms_of_use","contact","document","attachment"

```

Comment:
You're not talking about BetOnMarkets now are you? :stuck_out_tongue: (yes yes, byte order mark)